unstructured/partition/pdf.py

```diff
@@ -352,19 +351,133 @@ def extractable_elements(
     )
 
 
-def get_the_last_modification_date_pdf_or_img(
-    file: Optional[bytes | IO[bytes]] = None,
-    filename: Optional[str] = "",
-    date_from_file_object: bool = False,
-) -> str | None:
-    last_modification_date = None
-    if not file and filename:
-        last_modification_date = get_last_modified_date(filename=filename)
-    elif not filename and file:
-        last_modification_date = (
-            get_last_modified_date_from_file(file) if date_from_file_object else None
+def _partition_pdf_with_pdfminer(
```

Comment:
@christinestraub - Do you know why the diff for this file is so big?

Reply:
It's because I did some code refactoring in this PR.

Reply:
Sounds good! Will go through all the commits this morning.